src/fn/compile-app-settings.js

```diff
@@ -39,7 +40,43 @@ const compileAppSettingsForProject = async (projectDir, options) => {
   }
 
   const readOptionalJson = path => fs.exists(path) ? fs.readJson(path) : undefined;
-  const appSettings = fs.readJson(path.join(projectDir, 'app_settings.json'));
+  let appSettings;
+  const baseSettingsPath = path.join(projectDir, 'app_settings/base_settings.json');
+  const appSettingsPath = path.join(projectDir, 'app_settings.json');
+  if (!fs.exists(baseSettingsPath) && !fs.exists(appSettingsPath)) {
```

Comment:
this is a nit, but I'd add some empty lines between logical groups to improve readability.